Add documentation for collapse, oversample, truncate_hits processors (#5881)
opensearch-project/documentation-website: _search-plugins/search-pipelines/collapse-processor.md

---
layout: default

> title: Collapse processor

Reviewer: ```suggestion
title: Collapse
```

nav_order: 7
has_children: false
parent: Search processors
grand_parent: Search pipelines
---

# Collapse processor

> The `collapse` response processor discards hits that have the same value for some field as a previous document in the result set.

Reviewer: ```suggestion
The `collapse` response processor discards hits that have the same value for a particular field as a previous document in the result set.
```

> This is similar to the `collapse` parameter that can be passed in a search request, but the response processor is applied to the

Reviewer: ```suggestion
This is similar to passing the `collapse` parameter in a search request, but the response processor is applied to the
```
